src/main/java/de/tum/in/www1/artemis/domain/plagiarism/PlagiarismComparison.java

```diff
@@ -16,7 +18,7 @@
  */
 @Entity
 @Table(name = "plagiarism_comparison")
-public class PlagiarismComparison<E extends PlagiarismSubmissionElement> extends DomainObject {
+public class PlagiarismComparison<E extends PlagiarismSubmissionElement> extends DomainObject implements Comparable {
```

Comment:
```suggestion
public class PlagiarismComparison<E extends PlagiarismSubmissionElement> extends DomainObject implements Comparable<PlagiarismComparison<E>> {
```

Reply:
Thanks for the hint, I was trying something similar but unfortunately failed